alembic/script.py.mako

```diff
@@ -16,6 +17,19 @@ from alembic import op
 import sqlalchemy as sa
 ${imports if imports else ""}
 
+
+try:
+    is_sqlite = op.get_context().dialect.name == 'sqlite'
+except:
+    is_sqlite = False
```

Comment:
Our tests are the only place, yes. But, without it we wouldn't be able to fail on missing migrations.